build/README.md

```diff
@@ -895,6 +896,80 @@ Note the version in the pathname. Go may eliminate the need to do this in future
 
 We also use vendor to hold code patches while waiting for the project to release the fixes in their own code. An example is in [k8s.io/apimachinery](https://github.com/googleforgames/agones/issues/414) where a fix will be released later this year, but we updated our own vendored version in order to fix the issue sooner.
 
+## Running Performance Test
+
+To be able to run the performance script located in the following path: `agones/build/performance-test.sh` you need to have the following setup:
+
+#### Install a standard GKE cluster 
+
+Follow for more details here: https://agones.dev/site/docs/installation/creating-cluster/gke/
+
+#### Install Agones
+
+Follow for more details here: https://agones.dev/site/docs/installation/install-agones/helm/
+
+#### Set up node for prometheus 
+
+```
+gcloud container node-pools create agones-metrics --cluster={CLUSTER_NAME} --zone={REGION} \
+  --node-taints agones.dev/agones-metrics=true:NoExecute \
+  --node-labels agones.dev/agones-metrics=true \
+  --num-nodes=1 \
+  --machine-type=e2-standard-4
+```
+
+#### Install Prometheus, Grafana and port-forward
+
+cd agones/
+
+https://agones.dev/site/docs/guides/metrics/#installation
+
+#### Modify the performance-test.sh
+
+The performance-tests script contains a set of variables that need to be overwritten to work with your cluster and configuration settings. You can also pass these values through command line This is an example:
+
+```
+CLUSTER_NAME=agones-standard
+CLUSTER_LOCATION=us-central1-c
+REGISTRY=us-east1-docker.pkg.dev/user/agones
+PROJECT=my-project
+REPLICAS=10000
+AUTO_SHUTDOWN_DELAY=60
+BUFFER_SIZE=9900
+MAX_REPLICAS=20000
+DURATION=10m
+CLIENTS=50
+INTERVAL=1000
+```
+
+You might also want to comment out the first couple lines that come after the variables are set and also change the cd directoy:
+
+```
+# export SHELL="/bin/bash"
+# mkdir -p /go/src/agones.dev/
+# ln -sf /workspace /go/src/agones.dev/agones
+# cd /go/src/agones.dev/agones/build
+
+# gcloud config set project $PROJECT
+# gcloud container clusters get-credentials $CLUSTER_NAME \
+#        --zone=$CLUSTER_LOCATION --project=$PROJECT
+
+make install LOG_LEVEL=info REGISTRY='"'$REGISTRY'"' DOCKER_RUN=""
```

Comment:
This should also be commented out, right?

Reply:
True